### PR TITLE
Kernhanda halide header

### PR DESCRIPTION
**Pull request by abadams**
Opening #2839 from inside the repo for testing

### CODE REVIEW

**Comment by abadams** [comment]
Tests pass, someone PTAL

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -394,9 +394,9 @@ set(HEADER_FILES
 file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/include")
 file(TO_NATIVE_PATH "${CMAKE_BINARY_DIR}/include/" NATIVE_INCLUDE_PATH)
 add_custom_command(OUTPUT "${CMAKE_BINARY_DIR}/include/Halide.h"
-  COMMAND build_halide_h ${HEADER_FILES} HalideFooter.h > "${NATIVE_INCLUDE_PATH}Halide.h"
+  COMMAND build_halide_h ../LICENSE.txt ${HEADER_FILES} > "${NATIVE_INCLUDE_PATH}Halide.h"
```
Comment (steven-johnson) [inline src/CMakeLists.txt]: these two might be better as `${SRC_DIR}/../LICENSE.txt` or some other absolute-ish path

**Comment by steven-johnson** [comment]
LGTM

**Comment by abadams** [comment]
I need an actual github approval so I can hit merge :)

**Comment by kernhanda** [comment]
I don't think I can make updates to this branch to make the change @steven-johnson suggested.

**Comment by abadams** [comment]
I sent you an invite so that you can.

**Comment by kernhanda** [comment]
Made the suggested changes.

**Comment by kernhanda** [comment]
mingw buildbot seems to be having issues?

```shell
tar -czf distrib/halide.tgz \
	halide/bin \
	halide/lib \
	halide/include \
	halide/tutorial \
	halide/BUILD \
	halide/README*.md \
	halide/README_bazel.md \
	halide/WORKSPACE \
	halide/*.bzl \
	halide/*.cmake \
	halide/tools/mex_halide.m \
	halide/tools/*.cpp \
	halide/tools/halide_benchmark.h \
	halide/tools/halide_image.h \
	halide/tools/halide_image_io.h \
	halide/tools/halide_image_info.h
bash: tar: command not found
make: *** [../halide/Makefile:1762: distrib/halide.tgz] Error 127
```

Is there a way to retrigger the failed build manually? Or does a new commit need to be pushed?

**Comment by steven-johnson** [comment]
One of the buildbots might be missing tar. Let me check.

**Comment by steven-johnson** [comment]
Yep, winbot1 was missing it. Just queued a rebuild.

**Pull request merged**